public/app/features/dashboard/components/PanelEditor/state/actions.ts

```diff
@@ -58,6 +58,7 @@ export function panelEditorCleanUp(): ThunkResult<void> {
         if (lastResult) {
           sourcePanel.getQueryRunner().pipeDataToSubject(lastResult);
         }
+        sourcePanel.refresh();
```

Comment:
We should not tigger a refresh here, the code above will make sure the panel get's latest data again. 

This could be needed to clear alert state, but think that is an edge case.

Reply:
Without the refresh the sourcePanel will look like it still has an alert rule with the hear icon for instance.

Reply:
Any suggestions on how to force update the source panel @torkelo ?

Reply:
In graph render maybe we can check if we have alertState on the data but no alert rule defined, and if that is the case ignore ore remove alert state from data

Reply:
https://github.com/grafana/grafana/blob/master/public/app/features/dashboard/dashgrid/PanelChromeAngular.tsx#L98 

If we there can check if panel.alert is stiill defined